app/views/layouts/listnav/_container_link_to.html.haml

```diff
@@ -0,0 +1,43 @@
+- return if object.nil?
+- if object.kind_of?(ActiveRecord::Relation)
+  -# has_many relation
+  - klass = object.klass.base_model
+  - if klass == Container
+    - feature = 'containers'
+  - else
+    - feature ||= "#{klass.name.underscore}_show_list"
+  - label = ui_lookup(:models => klass.name)
+  - if role_allows(:feature => feature)
+    - count = object.count
+    - if count == 0
+      %li.disabled
+        = link_to(_("#{label} (#{count})"), "#")
+    - elsif count == 1
+      - object = object.first
+      - klass = object.class.base_model
+      %li
+        = link_to(_("#{label} (#{count})"),
+                  {:controller => "#{klass.name.underscore}", :action => 'show', :id => object},
+                  :title => _("View #{label} #{object.name}"))
+    - else
+      %li
+        = link_to(_("#{label} (#{count})"),
```

Comment:
I know there's an open request for clarification on details of the relationship between `_` and `ui_lookup`, etc... but including `#{count}` here is All Sorts Of Wrong.